_articles/tensorman.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: article
+title: Use Tensorman
+description: >
+  Using TensorMan to manage TensorFlow Containers
+keywords:
+  - NVIDIA
+  - CUDA
+  - TensorFlow
+  - TensorMan
+image: http://support.system76.com/images/system76.png
+hidden: false
+section: software-applications
+
+---
+
+# Using Tensorman
+
+<u>Tensorman</u> is a new tool for managing TensorFlow tool-chains in Pop!\_OS 19.10 (and coming soon to Pop!\_OS 18.04 LTS). It can be installed with this command:
+
+```
+sudo apt install tensorman
+```
+
+For NVIDIA CUDA support the following package must be installed:
+
+```
+sudo apt install nvidia-container-runtime
+```
+
+The user account working with Tensorman must be added to the `docker` group if that hasn't been done already:
+
+```
+sudo usermod -aG docker $USER
+```
+
+If Docker was just installed, log out and back in before using Tensorman.
```

Comment:
A log out may not be enough on 19.10, due to systemd user sessions. Reboot is better.

Reply:
Fixed.